ui/helpers/utils/transactions.util.js

```diff
@@ -225,3 +225,10 @@ export function getTransactionTypeTitle(t, type, nativeCurrency = 'ETH') {
  */
 export const isNFTAssetStandard = (assetStandard) =>
   assetStandard === ERC1155 || assetStandard === ERC721;
+
+export function formatWallet(wallet) {
```

Comment:
This does not seem to be used anywhere anymore. Do we still need it? If not can we remove it?